sig-aws/kubernetes-on-aws.md

```diff
@@ -0,0 +1,19 @@
+Kubernetes on Amazon Web Services
+====
+
+This page lists different options of starting a Kubernetes cluster on Amazon Web Services.
+
+* Clocker: http://www.clocker.io/tutorials/kubernetes-cluster.html
```

Comment:
The list was already in alphabetical order, just updated introductory text to clarify that. 

Did you mean something else?